src/jobsautoreport/models.py

```diff
@@ -13,5 +14,21 @@ class JobState(Enum):
     SUCCESS = "success"
     FAILURE = "failure"
 
+class EquinixUsage(BaseModel):
+    description: Optional[str]
```

Comment:
I see only a subset of these parameters being specified in the doc: https://deploy.equinix.com/developers/api/metal#tag/Usages/operation/findProjectUsage

where are the other fields coming from?

Reply:
I get also `metro`, `start_date` and `end_date` in the response, maybe they are optional